containers/foreman.yml

```diff
@@ -10,6 +10,16 @@
     pulp_worker_count: 2
     registry: projgriffin
   tasks:
+    - set_fact:
+        deployment_state: present
+      tags:
+        - start
+        - stop
```

Comment:
I think this list should include `restart` ?

Reply:
:+1: Done